mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -2195,6 +2214,128 @@ impl MmCoin for TendermintCoin {
     fn on_disabled(&self) -> Result<(), AbortedError> { AbortableSystem::abort_all(&self.abortable_system) }
 
     fn on_token_deactivated(&self, _ticker: &str) {}
+
+    async fn handle_balance_stream(self, ctx: MmArc) {
+        fn generate_subscription_query(query_filter: String) -> String {
+            let q = json!({
+                "jsonrpc": "2.0",
+                "method": "subscribe",
+                "id": 0,
+                "params": {
+                    "query": query_filter
+                }
+            });
```

Comment:
I think it's worth using a dedicated struct here instead of `json` macro. Possible candidate is
`jsonrpc_core::request::MethodCall`, but maybe we can use something from `tendermint_rpc` crate?

Reply:
`tendermint_rpc` seems too high level, I couldn't find a way to do what we do here. Used `MethodCall` instead as it provides what exactly we want https://github.com/KomodoPlatform/komodo-defi-framework/pull/1978/commits/3c50310ec5b0b795eeaea8e5480965e015aba909.